src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -215,9 +215,12 @@ internal void AddProject(AbstractProject project)
 
         internal void StartPushingToWorkspaceAndNotifyOfOpenDocuments(IEnumerable<AbstractProject> projects)
         {
-            foreach (var hostState in _workspaceHosts)
+            using (Dispatcher.CurrentDispatcher.DisableProcessing())
```

Comment:
I considered that, but I thought the HostReady check was actually what this was trying to override.  I'll dig deeper to understand if it needs the HostReady stuff.